ethereum/ethereum-org-website
# fix: add missing wallet image for Taiko Mainnet

## Description
Fixes #14530

This PR resolves the missing wallet image issue on the Find Wallet page by adding the required network data for Taiko Mainnet in the network configuration.

### Changes Made
- Added Taiko Mainnet network configuration in `src/data/networks/network.ts`
- Fixed missing wallet image display on the Find Wallet page

### Screenshots
Before 
![image](https://github.com/user-attachments/assets/69a0a33f-9287-4a6e-9686-79e79e186ef8)
After
![image](https://github.com/user-attachments/assets/599b4dc2-4ce5-449d-a846-92ae877b419c)


### Testing Instructions
1. Navigate to https://ethereum.org/en/wallets/find-wallet/
2. Scroll to the wallet section and find MetaMask
3. Verify that the previously missing image is now displayed correctly

### Additional Notes
- Follows existing network configuration pattern
- No breaking changes introduced
